packages/bluebird/src/abstracts/_shadows.scss

```diff
@@ -1,6 +1,15 @@
 // Shadows
 // ------------------------------------ *
-$depth1: 0 1px 0 0 rgba(185, 200, 212, 0.5); //border: 1px solid #D8DBE1;
-$depth2: 0 3px 6px 0 rgba(183, 183, 183, 0.5), 0 1px 0 0 rgba(223, 223, 223, 0.5); //border: 1px solid #BFCFDB;
-$depth3: 0 37px 60px 10px #D7E0E5;
-$depth4: 0 0 0 1px rgba(63, 63, 68, 0.05), 0 1px 3px 0 rgba(63, 63, 68, 0.15);
+$depth1: 0 1px 0 0 rgba(185, 200, 212, 0.5); //deprecated 
```

Comment:
Future Bluebird story: add `!default` tag to our root abstracts to allow for easier theming/overriding.

Reply:
Added this story to our backlog.